Content.Server/GameObjects/EntitySystems/MoverSystem.cs

```diff
@@ -78,6 +86,8 @@ public override void Initialize()
             SubscribeEvent<PlayerDetachedSystemMessage>(PlayerDetached);
 
             _audioSystem = EntitySystemManager.GetEntitySystem<AudioSystem>();
+
+            _configurationManager.RegisterCVar("game.pixelmovement", false, CVar.ARCHIVE);
```

Comment:
I know this is probably just for testing, but this should default to true (for the SS14 content pack)

Reply:
You are correct, it's only for testing